tests/test_range.py

```diff
@@ -149,5 +150,157 @@ def test_pickle(self):
             self.assertEqual(r1, r2)
             self.assertEqual(type(r1), type(r2))
 
+    def test_mereology(self):
+        for _ in range(1000):
+            d1 = DateRange.DATE_TYPE.fromordinal(random.randint(0, 1000000))
```

Comment:
unittests should be deterministic.
However, you may use `random`, as long as you initialize it with a hardcoded seed (to make it deterministic).